### PR TITLE
Routes header footer

### PR DESCRIPTION
**Pull request by sanu29**

### CODE REVIEW

File: src/components/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    lastname:"none",
+    email:"none",
+    password:"none"
+
+  }) 
+  const [show, setShow] = useState(false);
+  const handleClick = () => setShow(!show);
+  const navigate = useNavigate();
+  const [error, setError] = useState("none")
+  console.log(error)
```
Comment (aiman-ansari) [inline src/components/Signup.jsx:22]: remove console log

File: src/components/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                 onClick={handleClick}
+                 >
+                    {show ? 'Hide' : 'Show'}
+                </Button>
+            </InputRightElement>
+        </InputGroup>
+            <Text pt={'1rem'}></Text>
+             <Button bgColor={"teal.300"} color={"teal.900"}  fontWeight={"bold"} onClick={()=>{
+               const validation = SignUpValidation(userDetails)
+               console.log(validation)
```
Comment (aiman-ansari) [inline src/components/Signup.jsx:73]: here also

**Review by sanjitsarkar — APPROVED** [review]
Looks good to merge

**Review by sanjitsarkar — APPROVED** [review]
Looks good to merge

**Review by aiman-ansari — APPROVED** [review]
Good folder structure. Naming is also declarative .
Remove console log and use prettier for formatting your code.

**Pull request merged**